src/power_grid_model_io/converters/base_converter.py

```diff
@@ -121,10 +125,13 @@ def save(
           extra_info: Optional[ExtraInfoLookup]:  (Default value = None)
           destination: Optional[BaseDataStore[T]]:  (Default value = None)
 
+        Keyword Args:
+          data_type: Optional[str]:   (Default value = None) Output data type "sym_output" or "asym_output"
```

Comment:
This is only true for the pandapower output. The keywords `sym_output` and `asym_output` are actually borrowed from power grid model, but you're using it to specify the pandapower output format. If woul dbe easy to check if a pgm attibute is sym (1 value) or asym (3 values), so you don't need an extra attribute for it.

Reply:
For example, the PgmJsonConverter doesn't care if the output data is sym or asym, it is written a way that it will convert pgm data to the corresponding json structure in any case.